script/bootstrap

```diff
@@ -14,6 +14,14 @@ if [ -z "$GOPATH" ]; then
     ln -s "$GOPATH" src/github.com/git-lfs/git-lfs
 fi
 
+if uname -s | grep -q "_NT-"; then
+  echo "Installing goversioninfo to embed resources into Windows executables..."
```

Comment:
Do we have to do this every time `script/bootstrap` is run? I think we should wrap this in a:

```sh
if [ ! -d "$GOPATH/src/github.com/josephspurrier/goversioninfo" ]; then
  echo "Installing goversioninfo ..."
  go get github.com/josephspurrier/goversioninfo/cmd/goversioninfo
fi

export PATH="$GOPATH/bin/windows_386:$PATH"
echo "Creating the resource.syso version information file..."

go generate
```